### ISSUE TITLE
Title command param has wrong highlighting.

### ISSUE THREAD

**Issue by Panossa** [42iscool42/MCC#24]
See attached image:
![snap_2017 07 12_01h56m04s_013_](https://user-images.githubusercontent.com/23323875/28095911-6232783a-66a5-11e7-9c4a-1fb0e3e2f81d.png)

Example Code:
```
title @a title {"text":"something"}
title @a subtitle {"text":"something"}
```

**Comment by Panossa** [comment]
It's basically like with /recipe give and take. But recipe got fixed.

**Comment by 42iscool42** [comment]
This is something that can not be fixed in the current system the syntax highlighter works.  A new version is in the works that should have command specific highlighting that will cover issues like this.

**Comment by 42iscool42** [comment]
This has been fixed in the beta 1.13 version.
